README.md

```diff
@@ -2,46 +2,97 @@
 
 Monitor your Heimdall, Bor and Virtual machine node with Prometheus and Grafana
 
-## Follow below steps to set up monitoring on Heimdall and Bor
+## Follow below steps to set up monitoring for you Matic Node
+
+**Pre-Requesites**: You need to make sure that you Matic Node is setup, including Heimdall and Bor nodes. 
 
 **Step 1:**
 
-Enable prometheus flag to true on heimdall `config.toml` file.
+You will need to change `prometheus` flag to `true` in the `config.toml`. 
 
-```
-vim $HOME/.heimdalld/config.toml
+To access the config.toml here:
 
-# change prometheus flag to true
+**Linux Packages**: `sudo vi /etc/heimdall/config/config.toml`
+**Binaries**: `vi ~/.heimdalld/config/config.toml`
 
-prometheus = true
-```
+By default the prometheus flag is set to `false`.
+
+Make sure that you keep the formatting intact.
 
 **Step 2:**
 
-You need to restart the heimdall node since config change
+Once you have made the changes in the `config.toml` you will need to stop Heimdall service and restart it.
+
+To stop heimdall service:
+
+**Linux Packages**: `sudo service heimdalld stop`
+**Binaries**: `pkill heimdalld`
+
+To restart heimdall service:
+
+**Linux Packages**: `sudo service heimdalld start`
+**Binaries**: `heimdalld start`
 
 **Step 3:**
-```
-start bor with --metrics --pprof --pprofport 7071 --pprofaddr 0.0.0.0
-```
+
+You will make changes in the Bor service file. To access the `bor.service` file you can run this command:
+
+`locate bor.service`
+
+There would be multiple entries. You will need to edit the file in this system path: `/etc/systemd/system/bor.service`
+
+To edit the service file you can run the following command:
+
+`sudo nano /etc/systemd/system/bor.service`
+
+Now in this you would see `ExecStart=/bin/bash` with multiple paramaters in line to it. You will need to search for `--maxpeers 150`. Take your cursor to this paramater and then add this, `--metrics --pprof --pprofport 7071 --pprofaddr 0.0.0.0`
```

Comment:
You dont need to add it there itself compulsorily. It can be added anywhere. Maybe you can say - for e.g. I added it after `--maxpeers 150`

Reply:
Done